### PR TITLE
Clarify [removed] and [deleted].

### PR DESCRIPTION
**Pull request by minecast**
Make these terms more explicit in order to reduce user confusion.

### CODE REVIEW

**Comment by 13steinj** [comment]
There's literally no point to this. It's understandable enough, and I'll bet you my life savings you won't be able to find someone who actually is confused, (not literally, since there are some individuals out there, but you know what I mean). Even if one was confused. It's extremely easy to explain.

Also, this is not done for submissions, you only edited the comment text.

**Comment by minecast** [comment]
To the average user, "removed" and "deleted" sound pretty much the same, I mean unless you explain it to them, most aren't going to know the difference. It is perfectly acceptable to say "The user removed the comment" or "The moderator deleted the comment". I don't really see any harm in doing it.

Also, I fixed that thing you mentioned.

**Comment by 13steinj** [comment]
It sounds nothing alike to me, nor to the "average user". Prove it. I see no harm, but I also see no gain.

Also...you didn't fix it for posts. Look at the logic. That only occurs if a post is both deleted and removed. And, even if you did it logically sound, doing it in `add_props` would break some technical things that are related to the use of the `expunged` attribute and it's boolean logic. To edit the text on posts, you'd need to make a change at least around [here](https://github.com/reddit/reddit/pull/1431/files), maybe somewhere else. But not in `add_props` AFAIK.

**Comment by Pokechu22** [comment]
> It sounds nothing alike to me, nor to the "average user". Prove it. I see no harm, but I also see no gain.

I'll agree with @minecast -- it does seem rather similar.  Unless you see two next to each other, you'll have trouble noticing the difference.

It's not necessarily the average user that'll have an issue (or particularly care), it's more important to make it clear to those who would assume bad faith if they saw a missing comment.  Fine, some of those users will know the difference, but it's not obvious.

If a brand new user saw a "removed" comment, they wouldn't know what that means in comparison to "deleted".  All of us involved in this conversation understand the significance, but most people won't.

**Comment by 13steinj** [comment]
@Pokechu22 *shrug*. Never actually seen an issue come up regarding it or it's "confusion" in all my time.

File: r2/r2/lib/jsontemplates.py [hunk 1]
```diff
@@ -685,7 +685,7 @@ def thing_attr(self, thing, attr):
             if not thing.expunged:
                 return thing.selftext
             else:
-                return ''
+                return '[removed by moderator]'
         elif attr == 'selftext_html':
             if not thing.expunged:
                 return safemarkdown(thing.selftext)
```
Comment (13steinj) [inline r2/r2/lib/jsontemplates.py]: @minecast You forgot to do it on #L693

**Comment by english06** [comment]
Great pull request. Only mods really know the difference between "removed" and "deleted". This is a great clarification.

**Comment by minecast** [comment]
@english06 Hmm, yeah. I'll try adding that when I can.

**Comment by english06** [comment]
I am sadly the wrong guy to ask. Poking @13steinj for help.

**Comment by 13steinj** [comment]
@english06 Sorry, that's extremely strange. Both times you commented on something of mine / poked me I didn't get a notification.

@minecast No, it wouldn't, as `c.user_is_admin` is the state of the currently authenticated user being an admin, not the user that made the removal be an admin. You can not distinguish between admin and moderator removals, from a glance. Regardless, the admins never remove anything.

**Comment by Pokechu22** [comment]
@13steinj Well, they may occasionally remove things, but that's usually due to a DCMA notice and thus is logged in [/r/ChillingEffects](//reddit.com/r/ChillingEffects) and is [already marked as such](https://www.reddit.com/r/ChillingEffects/comments/35urvq/test_post_please_ignore/).  I agree that it doesn't seem too necessary to distinguish between admin and moderator in that message, though.

**Comment by 13steinj** [comment]
@Pokechu22 yeah, I just disregarded the DMCA removals as those are completely different.

**Comment by english06** [comment]
I certainly agree that they most likely rarely, if ever, act in that manner. I just wanted to account for it if possible. If not, I think the "removed by moderator" vs "deleted by user" clarification is very valuable in itself.

**Comment by minecast** [comment]
Closing this and starting new PR with all changes in 1 PR due to my stupidity

**Pull request closed without merging**